src/azul/openapi/validation.py

```diff
@@ -0,0 +1,187 @@
+import json
+import logging
+from more_itertools import one
+import re
+from typing import (
+    Dict,
+    List,
+    Mapping,
+)
+from chalice import (
```

Comment:
One blank line needed between first group of Python-bundled modules and second group of external modules.